src/app/_components/CrossIcon/CrossIcon.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function PlusIcon() {
```

Comment:
🟢  can you please replace default export with named?

Reply:
Thanks for comment! Done.